Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -163,6 +163,14 @@ public extension CBPurchase {
             }
         }
     }
+    
+    func showManageSubscriptionsSettings() {
+        if #available(iOS 15.0, *) {
+            UIApplication.showManageSubscriptions()
+        } else {
+            // Fallback on earlier versions
```

Comment:
1. How do we want to handle this for earlier versions?
2. Should this method be accessed via`CBPurchase` or `Chargebee`?

Reply:
Hi i have updated this Fall backversion, it will open the Mange subscriptions in Appstore App setting for lowerversion

Reply:
Can you address the 2nd point as well?

Reply:
I think we should add it in Chargebee, not CBPurchase. I will update it!

Reply:
Addressed from CBPurchase